packages/dapi/test/unit/externalApis/drive/fetchProofForStateTransitionFactory.spec.js

```
@@ -16,6 +16,8 @@ const getDocumentsFixture = require('@dashevo/wasm-dpp/lib/test/fixtures/getDocu
 const getBlsAdapterMock = require('@dashevo/wasm-dpp/lib/test/mocks/getBlsAdapterMock');
 const createStateRepositoryMock = require('@dashevo/wasm-dpp/lib/test/mocks/createStateRepositoryMock');
 const fetchProofForStateTransitionFactory = require('../../../../lib/externalApis/drive/fetchProofForStateTransitionFactory');
+const getBlsAdapterMock = require('@dashevo/wasm-dpp/lib/test/mocks/getBlsAdapterMock');
```

Biome (1.9.4)

[error] Shouldn't redeclare 'getBlsAdapterMock'. Consider to delete it or rename it.

'getBlsAdapterMock' is defined here:


(lint/suspicious/noRedeclare)